src/ceph-volume/ceph_volume/util/disk.py

```diff
@@ -786,6 +786,8 @@ def get_devices(_sys_block_path='/sys/block', _dev_path='/dev', _mapper_path='/d
     block_devs = get_block_devs(_sys_block_path)
     dev_devs = get_dev_devs(_dev_path)
     mapper_devs = get_mapper_devs(_mapper_path)
+    if lvs is None and lvs != []:
```

Comment:
Hmm this condition can never be true iiuc?

Reply:
You are right. Replaced it with `if lvs is None or len(lvs) == 0:`. Thanks!